recipe/meta.yaml

```diff
@@ -44,6 +44,8 @@ test:
   # https://github.com/conda-forge/staged-recipes/pull/292#issuecomment-208080893.
   commands:                # [not win]
     - nosetests zmq.tests  # [not win]
+  imports:                 # [win]
+    - zmq                  # [win]
```

Comment:
You are absolutely right. I'm not sure why this is broken, then, because downstream packages are failing to build on win+py35 because they can't `import zmq`.

https://github.com/conda-forge/ipyleaflet-feedstock/pull/6

If this ran before, I'm not sure what could be causing it to fail now. I just saw that the only failing case was win+py35 and that happens to also be a special-cased build here, so I figured it must be the culprit.